conda/cli/conda_argparse.py

```diff
@@ -97,42 +100,48 @@ def generate_parser():
     configure_parser_info(sub_parsers)
     configure_parser_init(sub_parsers)
     configure_parser_install(sub_parsers)
+    configure_parser_notices(sub_parsers)
     configure_parser_list(sub_parsers)
```

Comment:
I assume the intent here was for it to be alphabetical:

```suggestion
    configure_parser_list(sub_parsers)
    configure_parser_notices(sub_parsers)
```

Reply:
🤦